nsist/copymodules.py

```diff
@@ -73,8 +73,8 @@ def copytree_ignore_callback(excludes, pkgdir, modname, directory, files):
 
     # Filter by file names relative to the build directory
     reldir = os.path.relpath(directory, pkgdir)
-    target = os.path.join('pkgs/', modname, reldir)
-    files = [os.path.join(target, fname) for fname in files]
+    target = os.path.join('pkgs' + os.path.sep, modname, reldir)
```

Comment:
os.path.sep should be redundant here - join() will add the correct separator automatically between path components.